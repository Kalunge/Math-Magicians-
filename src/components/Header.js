import React from 'react';
import { Link } from 'react-router-dom';
import {BrowserRouter as Router} from 'react-router-dom'
const Header = () => {
  const style = {
    color: 'blue',
  };
  return (
    <nav className="navBar">
      <div>
        <h1>Math Magicians</h1>
      </div>
      <ul style={style}>
        <Router>
        <Link to="/" className="link home">
          <li>Home</li>
        </Link>
        <Link to="/calculator" className="link calc">
          <li>Calculator</li>
        </Link>
        <Link to="/quote" className="link quote">
          <li>Quote</li>
        </Link>
        </Router>
      </ul>
    </nav>
  );
};

export default Header;
