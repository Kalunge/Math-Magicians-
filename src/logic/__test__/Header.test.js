import React from 'react';
import { expect } from '@jest/globals';
import renderer from 'react-test-renderer';

import Header from '../../components/Header';

describe('test header', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot('Math Magicians');
  });
});