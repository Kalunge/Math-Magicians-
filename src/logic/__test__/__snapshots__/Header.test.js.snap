// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header renders correctly: Math Magicians 1`] = `
<nav
  className="navBar"
>
  <div>
    <h1>
      Math Magicians
    </h1>
  </div>
  <ul
    style={
      Object {
        "color": "blue",
      }
    }
  >
    <a
      className="link home"
      href="/"
      onClick={[Function]}
    >
      <li>
        Home
      </li>
    </a>
    <a
      className="link calc"
      href="/calculator"
      onClick={[Function]}
    >
      <li>
        Calculator
      </li>
    </a>
    <a
      className="link quote"
      href="/quote"
      onClick={[Function]}
    >
      <li>
        Quote
      </li>
    </a>
  </ul>
</nav>
`;
